Hestia/Hestia/frontend/DevicesScreen/AddDeviceScreen/PropertyCell.cs

```diff
@@ -2,12 +2,14 @@
 using Foundation;
 using CoreGraphics;
 
-
 namespace Hestia.DevicesScreen.AddDeviceScreen
 {
     public class PropertyCell : UITableViewCell
     {
         public UITextField inputField;
+        const int IndentationLeft = 15;
+        const int IndentationTop = 4;
```

Comment:
Does this work well on every sort of iPhone size?

Reply:
Yes, it does. This way of laying out subviews is also done in the official Xamarin documentation https://docs.microsoft.com/en-us/xamarin/ios/user-interface/controls/tables/customizing-table-appearance